layouts/in-the-news.hbs

```diff
@@ -25,16 +26,16 @@
       window.onload = function () {
```

Comment:
We use the standard.js style across the nodejs.org website, please make sure client side scripts follow the same style.